addons/stock/models/stock_quant.py

```diff
@@ -122,8 +124,16 @@ def _compute_inventory_date(self):
     @api.depends('inventory_quantity')
     def _compute_inventory_diff_quantity(self):
         for quant in self:
+            quant.inventory_quantity_set = True
             quant.inventory_diff_quantity = quant.inventory_quantity - quant.quantity
 
+    @api.depends('inventory_quantity', 'quantity')
+    def _compute_if_outdated(self):
+        for quant in self:
+            quant.outdated = False
+            if fields.Float.compare(quant.inventory_quantity - quant.inventory_diff_quantity, quant.quantity, precision_rounding=quant.product_uom_id.rounding) and quant.inventory_quantity_set:
```

Comment:
As `float_compare` is already imported in the file, pourquoi s'embêter ? :man_shrugging: 
```suggestion
            if float_compare(quant.inventory_quantity - quant.inventory_diff_quantity, quant.quantity, precision_rounding=quant.product_uom_id.rounding) and quant.inventory_quantity_set:
```

Reply:
better to use fields.Float.compare now it exists.